### PR TITLE
Add arg for constant pad value in read.from_np

### PR DESCRIPTION
**Pull request by thomasdorch**
Added an optional argument to `read.from_np` to allow user to specify a value to pad the image array with.

### CODE REVIEW

File: gdsfactory/read/from_np.py [hunk 1]
```diff
@@ -42,13 +43,19 @@ def from_np(
         layer: layer tuple to output gds.
         threshold: value along which to find contours in the array.
         invert: invert the mask.
+        outer_pad_value: set outer padding used by np.pad to this constant value (optional).
```
Comment (joamatab) [inline gdsfactory/read/from_np.py:46]: is this in um or pixels?

how about calling it padding? we already have the concept of padding, which is in um,

some ideas are

- padding (if it's um)
- padding_pixels (if it's in pixels)

**Review by joamatab — CHANGES_REQUESTED** [review]
Thank you Thomas!

**Comment by thomasdorch** [comment]
@joamatab I made the description more specific to indicate that the value is used to set the border pixel values but let me know if something else needs to change for this to get merged

**Comment by joamatab** [comment]
I think the variable name could be a bit better

How about `outer_pad_pixels`?

**Comment by thomasdorch** [comment]
@joamatab Thanks for the feedback! I added an arg for the number of pixels to pad with as well and changed the names to hopefully more clear. Let me know if this is good

Also, not sure what happened with my rebase initially I fixed it now and it should be just the 3 commits.

**Comment by joamatab** [comment]
awesome! thank you Thomas!

**Review by joamatab — APPROVED** [review]

**Pull request merged**